atc/exec/set_pipeline_step.go

```diff
@@ -94,6 +94,15 @@ func (step *SetPipelineStep) run(ctx context.Context, state RunState) error {
 	fmt.Fprintln(stderr, "\x1b[33mfollow RFC #31 for updates: https://github.com/concourse/rfcs/pull/31\x1b[0m")
 	fmt.Fprintln(stderr, "")
 
+	if step.plan.Name == "self" {
+		fmt.Fprintln(stderr, "\x1b[1;33mWARNING: 'set_pipeline: self' is experimental and subject to change!\x1b[0m")
```

Comment:
Stronger scare-words might be a good idea so we can get more feedback:

```suggestion
		fmt.Fprintln(stderr, "\x1b[1;33mWARNING: 'set_pipeline: self' is experimental and may be removed in the future!\x1b[0m")
```